runtime/io/io.cpp

```diff
@@ -19,7 +20,7 @@ extern "C" {
   static block * dotK = (block *)((((uint64_t)getTagForSymbolName("dotk{}")) << 32) | 1);
   static blockheader kseqHeader = {getBlockHeaderForSymbol((uint64_t)getTagForSymbolName("kseq{}"))};
 
-  static std::map<std::string, std::string> logFiles;
+  static std::map<char *, std::string> logFiles;
```

Comment:
I'm not sure this works. a std::map whose key is char * is going to compare strings with pointer equality, which is not what you want.